docs/how-to-build.md

```diff
@@ -64,8 +64,8 @@ To build a `purl` string from its components:
   segments:
 
   - Append '#' to the `purl`
-  - Strip the `subpath` from leading and trailing '/'
-  - Split this on '/' as segments
+  - Divide the `subpath` into segments using the path delimiter of your
```

Comment:
as in https://github.com/package-url/purl-spec/pull/458/files#diff-9e903c2e2214758fe0d311b17101770605a41d5348217b9733ae4e778a2708d9